apps/frontend/src/components/overview/MemoryCard.tsx

```
@@ -16,28 +18,41 @@ export default function MemoryCard({
           className="object-cover rounded-lg"
         />
       ) : (
-        <div className="relative">
+        <div className="relative min-h-[400px] flex items-center justify-center">
           <img
             src="/no-memories-placeholder.jpg"
             alt=""
             role="presentation"
```

Biome (2.1.2)

[error] Using the role attribute 'presentation' on the 'img' element is redundant, because it is implied by its semantic.

Unsafe fix: Remove the role attribute.


(lint/a11y/noRedundantRoles)